FirebaseAdmin/FirebaseAdmin/Auth/ImportUserRecordArgs.cs

```diff
@@ -96,40 +96,19 @@ internal bool HasPassword()
             return this.PasswordHash != null;
         }
 
-        internal ImportUserRequest ToImportUserRequest()
+        internal ImportUserRecordArgsRequest ToImportUserRequest()
         {
-            return new ImportUserRequest(this);
+            return new ImportUserRecordArgsRequest(this);
         }
 
-        private static string UrlSafeBase64Encode(byte[] bytes)
+        internal sealed class ImportUserRecordArgsRequest
```

Comment:
May be just call this class `Request` so the full qualified name becomes `ImportUserRecordArgs.Request`. That's much more readable.